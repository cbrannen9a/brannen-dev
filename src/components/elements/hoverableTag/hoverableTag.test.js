import React from 'react';
import renderer from 'react-test-renderer';

import HoverableTag from './hoverableTag';

const title = 'Test Tag';
const blocks = [{
	style: 'normal', _key: 'block1', _type: 'block', markDefs: [],
	children: [
		{
			marks: [],
			text: 'test text',
			_key: 'mock1',
			_type: 'span'
		},
		{
			marks: ['strong'],
			text: 'test bold',
			_key: 'mock2',
			_type: 'span'
		},
		{
			marks: ['em'],
			text: 'test italic',
			_key: 'mock3',
			_type: 'span'
		}]
}];

describe('HoverableTag', () => {
	it('renders correctly', () => {
		const tree = renderer
			.create(<HoverableTag title={title} description={blocks} />)
			.toJSON();
		expect(tree).toMatchSnapshot();
	});
});