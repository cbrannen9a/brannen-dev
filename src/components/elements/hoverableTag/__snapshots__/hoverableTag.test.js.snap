// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HoverableTag renders correctly 1`] = `
<div
  style={
    Object {
      "marginRight": "4px",
    }
  }
>
  <span
    className="tooltip"
    onMouseLeave={[Function]}
  >
    <span
      className="tooltip-trigger"
      onMouseOver={[Function]}
    >
      <span
        className="tag"
      >
        Test Tag
      </span>
    </span>
  </span>
</div>
`;
